discover/faq.md

```diff
@@ -12,1638 +12,164 @@ Hopefully, you'll find the answers here! If not then please don't hesitate to re
-
-</details>
-
-<details>
-
-<summary>Why Publish?</summary>
-
-Publishing data, algorithms, and other digital assets and services on an Ocean-powered marketplace offers numerous opportunities to earn on the future revenue streams connected to that data as well as build a lucrative ecosystem that adds value to the published asset. It also allows for the discovery and insights into new use cases and applications of the published asset.
-
-</details>
-
-<details>
-
-<summary>What about the price fluctuation of Ocean?</summary>
-
-Price fluctuation is mitigated through the use of the Ocean backed stable coin H2O.
-
-</details>
-
-<details>
-
-<summary>Who pays for gas fees?</summary>
-
-Gas fees for marketplace transactions are paid by the user initiating the transaction (for publishing, consuming, etc).
-
-</details>
-
-<details>
-
-<summary>Where do the docker containers run?</summary>
-
-Dockers containers can run anywhere. Ocean Market uses a docker run by the Ocean Protocol Foundation OPF); limit: 1 CPU limit / 60 seconds max. NOTE: This means OPF technically has access to data. In the case of a forked Ocean-powered marketplace, the owner of the marketplace must set up a computation environment. If individual users of the marketplace are concerned with security they should be prepared to host both the data and provide compute-to-data services on-premise.
-
-</details>
-
-<details>
-
-<summary>Who pays for the computation?</summary>
-
-The marketplace owner.
-
-</details>
-
-<details>
-
-<summary>What cryptocurrency do I need for transactions?</summary>
-
-The type of cryptocurrencies needed for transactions on the marketplace depends on which network(s) the marketplace is running (Ethereum, Polygon, EWT, BSC, Moonriver, etc.). Regardless of network, users will need to have Ocean tokens as well as the corresponding network token, which is used to pay for gas.
-
-</details>
-
-<details>
-
-<summary>Can I use the off-the-shelf CSS available in the repo?</summary>
-
-The marketplace name, logo, and typeface must be changed by the client. A slight modification would be enough for compliance. For more information consult the READ ME file on [GitHub](https://github.com/oceanprotocol/market#-forking). 
-
-</details>
-
-<details>
-
-<summary>What kind of data can be sold on Ocean Protocol? Is it limited to trading, analysis, etc.?</summary>
-
-Any type of data can be sold via Ocean Protocol. AI data, music data, image, video, trading, tickets. Anything that can live behind a URL.
-
-</details>
-
-<details>
-
-<summary>What type of data can be sold via the Ocean Market? Can I sell an ebook through Ocean Protocol Market?</summary>
-
-Yes, any type of data can be sold via Ocean Protocol. Ai data, music, image, video, trading, tickets. Anything that is accessible online.
-</details>
-
-
-### Data Farming
```

Comment:
The DF, Staking and veOcean are refactored by @idiom-bytes in this PR: https://github.com/oceanprotocol/docs/pull/1261